entrypoint.sh

```diff
@@ -149,7 +153,7 @@ run_spot_instance () {
     --instance-market-options '{"MarketType":"spot", "SpotOptions": {"MaxPrice": "'${BID_PRICE}'" }}' \
     --block-device-mappings '[{"DeviceName": "/dev/sda1","Ebs": { "VolumeSize": '${ROOT_VOLUME_SIZE}', "DeleteOnTermination": true } }]'\
     $KEY_NAME_OPT \
-    --user-data file://user_data.sh)
+    --user-data file://user_data.sh 2>&1)
```

Comment:
2 >&1 will change log output stream, is this change save enough?

Reply:
we need to pick up the error message and parse the min bid price from the output, `2&1` can get the output